lib/frictionless_data_dataset.rb

```diff
@@ -214,6 +217,29 @@ def write_to_files(occurrence_ids)
             @problems << CSV::Row.new(problems_collector_header, data).to_s
           end
 
+          # Add missing_attributions.csv
+          uri = !o.u_orcid.nil? ? "https://orcid.org/#{o.u_orcid}" : "http://www.wikidata.org/entity/#{o.u_wikidata}"
+          if (o.action.include?("recorded") && !o.occ_recordedByID.split('|').map(|e| {e.strip}).include?(uri)) ||
```

Comment:
I just realised, it might be a faster here to skip all attributions that have `createdBy == 'GBIF Sourced'` before checking if it's in `occ_recordedByID` or `occ_identifiedByID` column